pkg/chunk/chunk_store.go

```diff
@@ -61,8 +61,8 @@ type StoreConfig struct {
 
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (cfg *StoreConfig) RegisterFlags(f *flag.FlagSet) {
-	cfg.ChunkCacheConfig.RegisterFlagsWithPrefix("", "Cache config for chunks. ", f)
-	f.BoolVar(&cfg.chunkCacheStubs, "store.chunk-cache-stubs", false, "If true, don't write the full chunk to cache, just a stub entry.")
+	cfg.ChunkCacheConfig.RegisterFlagsWithPrefix("store.chunks-cache.", "Cache config for chunks. ", f)
```

Comment:
This setting is disabled in YAML with the comment:
```
// Not visible in yaml because the setting shouldn't be common between ingesters and queriers
```

Could you explain me why? I can't understand the issue from the comment.

Reply:
Added comments explaining this